docs/source/tutorials/token_classification/token_classification.ipynb

```diff
@@ -0,0 +1,522 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "069a3ce3",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# %%capture \n",
+    "# !pip uninstall cleanlab -y \n",
+    "# !pip install git+https://github.com/ericwang1997/cleanlab.git@token_classification \n",
+    "# !pip install termcolor "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "33d67688",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# package versions: \n",
+    "# cleanlab==2.0.0 (ericwang/cleanlab -b token_classification)\n",
+    "# termcolor==1.1.0 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Token Classification Label Error Detection - Part 2 \n",
+    "\n",
+    "In this tutorial, we show how you can use cleanlab to find potential label errors in a NLP token classification dataset. Here, we use CONLL-2003 which contains 20,718 sentences and 301,361 tokens. In a standard NLP token-classification task, the model predicts the entity of each token within each sentence. In this example, the named entity includes miscellaneous `MISC`, location `LOC`, person `PER`, and organization `ORG`. Unnamed entities are labeled as `O`. \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "- Identify potential token label issues using cleanlab's `token_classification.filter.find_label_issues` method. \n",
+    "- Rank sentences using cleanlab's `token_classification.rank.get_label_quality_score` method. \n",
+    "- TODO: (Clean Learning) Train a more robust model by removing problematic sentences. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1da020bc",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a1349304",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from termcolor import colored \n",
```

Comment:
I don't think this line is necessary? where are you using `colored` function in this notebook?

Reply:
This is to highlight the tokens in the sentence (the red words)

Reply:
and right, I don't need it here in the tutorial (only in utils.py). Will remove it

Reply:
Let's keep the installation of the `termcolor` package in this notebook though. I'd like termcolor to remain an optional dependency of our package (will clarify details of how to make this happen in slack)

Reply:
Sorry disregard my old comment about keeping the installation of `termcolor` dependency in this notebook. Let's remove the pip install termcolor lines. 

Instead termcolor needs to become an official dependency of the cleanlab package.  So you should add it to here:
https://github.com/cleanlab/cleanlab/blob/master/setup.py#L92
and properly specify what is the oldest version of the termcolor package that will still work with cleanlab.